src/project/qml/MuseScore/Project/internal/ScoresPage/CloudScoresView.qml

```diff
@@ -21,6 +21,7 @@
  */
```

Comment:
This file looks too large. It would be great to move some code to separate components